astropy/io/asdf/setup_package.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license
+import os
+
+def get_package_data():
+    # Installs the schema files
+    schemas = []
+    root = os.path.join(os.path.dirname(__file__), 'schemas')
+    for node, dirs, files in os.walk(root):
+        for fname in files:
+            if fname.endswith('.yaml'):
+                schemas.append(
+                    os.path.relpath(os.path.join(node, fname), root))
+
+    # In the package directory, install to the subdirectory 'schemas'
+    schemas = [os.path.join('schemas', s) for s in schemas]
+
+    return {'astropy.io.asdf': schemas}
+
+
+def requires_2to3():
```

Comment:
Why is this needed?

Reply:
It's not; I think it was just a copy-paste error :)